com.flashforge.FlashPrint.appdata.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.flashforge.FlashPrint</id>
+  <name>FlashPrint</name>
+  <developer_name>Flashforge</developer_name>
+  <summary>Slicer for the FlashForge 3D printers</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=http://www.flashforge.com/terms-of-service/</project_license>
+  <url type="homepage">http://www.flashforge.com/flashprint/</url>
+  <description>
+    <p>FlashPrint is a beginner-friendly slicer developped in-house by Flashforge for use
+      with its line of 3D printers, featuring a 3D viewer, scaling and cutting abilities,
+      and multiple printer related configuration options. It also sports an expert mode for
+      more experienced users.</p>
```

Comment:
Can you put something like:
> NOTE: This wrapper is not verified by, affiliated with, or supported by FlashForge

here? It's important to make sure that users know that this isn't officially distributed for these sort of apps..